### PR TITLE
WSTEAM1-453: Add ATI tracking to live page

### PR DESCRIPTION
**Pull request by shayneahchoon**
Resolves [WSTEAM1-453](https://jira.dev.bbc.co.uk/browse/WSTEAM1-453)

Overall changes
======
Adds ATI tracking to live pages, sample network message:  
```s: 598343
idclient: 109fb2d9-54a1-422d-adec-8494ae7b2018
s2: 70
p: live_coverage.c07zr0zwjnnt.page
r: 1920x1080x24x24
re: 1300x938
hl: 11x39x31
lng: en-GB
x1: [urn:bbc:tipo:topic:c07zr0zwjnnt]
x2: [responsive]
x3: [news-pidgin]
x4: [pcm]
x5: [http%3A%2F%2Flocalhost%3A7081%2Fpidgin%2Flive%2Fc07zr0zwjnnt]
x6: [http%3A%2F%2Flocalhost%3A7081%2Fpidgin%2Flive%2Fc07zr0zwjnnt]
x7: [live-coverage]
x8: [simorgh]
x9: [Live%20experience%20pagination]
ref: http://localhost:7081/pidgin/live/c07zr0zwjnnt
```

Code changes
======

- _Adds ATI Tracking component to `/ws-nextjs-app/pages/[service]/live/[id]/LivePageLayout.tsx`._

Testing
======
1. _Go to `/pidgin/live/cdvxv61l6x8t`_
2. Go to Network Tab. 
3. Check that ATI Payload is being sent.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: ws-nextjs-app/tsconfig.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "next-env.d.ts",
     "**/*.ts",
     "**/*.tsx",
+    "../src/**/*.d.ts",
```
Comment (shayneahchoon) [inline ws-nextjs-app/tsconfig.json]: Hello, could someone let me know if this is okay?

File: src/app/contexts/EventTrackingContext/index.tsx [hunk 2]
```diff
@@ -88,15 +88,15 @@ export const EventTrackingContextProvider = ({
...
   const requestContext = useContext(RequestContext);
-  const { isNextJs, pageType } = requestContext;
+  const { pageType } = requestContext;
 
   const serviceContext = useContext(ServiceContext);
   const { atiAnalyticsProducerId } = serviceContext;
 
   const { enabled: eventTrackingIsEnabled } = useToggle('eventTracking');
 
   // TODO: Enable event tracking for NextJS pages
```
Comment (emilysaffron) [inline src/app/contexts/EventTrackingContext/index.tsx]: Can we remove this comment now this work is done?

File: ws-nextjs-app/tsconfig.json (hunk 1 above)
Comment (emilysaffron) [reply ws-nextjs-app/tsconfig.json]: Looks good, me and Aaron chatted about this as i had the same errors with the typing - this is the best way to let it use the express app type definitions!

**Review by emilysaffron — APPROVED** [review]

File: ws-nextjs-app/pages/[service]/live/[id]/LivePageLayout.tsx [hunk 3]
```diff
@@ -24,6 +25,12 @@ type ComponentProps = {
       content: StreamResponse | null;
       contributors: string | null;
     };
+    atiAnalytics: {
+      contentId: string;
+      contentType: string;
+      pageIdentifier: string;
+      pageTitle: string;
+    };
```
Comment (amoore108) [inline ws-nextjs-app/pages/[service]/live/[id]/LivePageLayout.tsx]: You may be able to use the type defined in https://github.com/bbc/simorgh/blob/latest/src/app/components/ATIAnalytics/types.ts#L67 for this

**Review by amoore108 — APPROVED** [review]

**Pull request merged**